### PR TITLE
HDFS-14522. Allow compact property description in xml in httpfs.

### PR DESCRIPTION
**Pull request by iwasakims**

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  23m  0s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 24s |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   0m 19s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 34s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  14m 27s |  branch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 21s |  trunk passed  |
| +0 :ok: |  spotbugs  |   0m 44s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 41s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 24s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 18s |  the patch passed  |
| -0 :warning: |  checkstyle  |   0m 13s |  hadoop-hdfs-project/hadoop-hdfs-httpfs: The patch generated 1 new + 8 unchanged - 5 fixed = 9 total (was 13)  |
| +1 :green_heart: |  mvnsite  |   0m 27s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  15m 36s |  patch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 20s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m 54s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   4m 39s |  hadoop-hdfs-httpfs in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 31s |  The patch does not generate ASF License warnings.  |
|  |   |  65m 48s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.fs.http.client.TestHttpFSFWithSWebhdfsFileSystem |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1737 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux bf9cb35d983e 4.15.0-70-generic #79-Ubuntu SMP Tue Nov 12 10:36:11 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 47fdae7 |
| Default Java | 1.8.0_222 |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/1/artifact/out/diff-checkstyle-hadoop-hdfs-project_hadoop-hdfs-httpfs.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/1/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-httpfs.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/1/testReport/ |
| Max. process+thread count | 596 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-httpfs U: hadoop-hdfs-project/hadoop-hdfs-httpfs |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/1/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  31m 48s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  20m 42s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 23s |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   0m 20s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 31s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  14m 36s |  branch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 21s |  trunk passed  |
| +0 :ok: |  spotbugs  |   0m 42s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 41s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 22s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 18s |  the patch passed  |
| -0 :warning: |  checkstyle  |   0m 14s |  hadoop-hdfs-project/hadoop-hdfs-httpfs: The patch generated 1 new + 7 unchanged - 5 fixed = 8 total (was 12)  |
| +1 :green_heart: |  mvnsite  |   0m 26s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedclient  |  15m 10s |  patch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m 45s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  |   4m 24s |  hadoop-hdfs-httpfs in the patch failed.  |
| +1 :green_heart: |  asflicense  |   0m 28s |  The patch does not generate ASF License warnings.  |
|  |   |  93m 48s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.fs.http.client.TestHttpFSFWithSWebhdfsFileSystem |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1737 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux a6c05fa1f144 4.15.0-66-generic #75-Ubuntu SMP Tue Oct 1 05:24:09 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 47fdae7 |
| Default Java | 1.8.0_222 |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/2/artifact/out/diff-checkstyle-hadoop-hdfs-project_hadoop-hdfs-httpfs.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/2/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs-httpfs.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/2/testReport/ |
| Max. process+thread count | 615 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-httpfs U: hadoop-hdfs-project/hadoop-hdfs-httpfs |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/2/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java [hunk 1]
```diff
@@ -124,4 +127,22 @@ public void testVarResolutionAndSysProps() {
     assertEquals(conf.get("user.name"), "foo");
   }
 
+  @Test
+  public void testCompactFormatProperty() throws IOException {
```
Comment (goiri) [inline hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java]: Should we add a test using a file in "resources" too?

Comment (iwasakims) [reply hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java]: The `ConfigurationUtils#load` takes only InputStream as input and does not care about where the data come from. I think loading configs from file is covered by test cases in o.a.h.lib.server.TestServer.

Comment (goiri) [reply hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java]: Having a file makes it cleaner than putting the whole string in code.
That being said, I think both should be there.

Comment (iwasakims) [reply hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java]: I updated the code based on your suggestion.

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 38s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 2 new or modified test files.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  27m 18s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 26s |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   0m 21s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 35s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  17m 38s |  branch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 25s |  trunk passed  |
| +0 :ok: |  spotbugs  |   0m 50s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 48s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 26s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 21s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 21s |  the patch passed  |
| -0 :warning: |  checkstyle  |   0m 15s |  hadoop-hdfs-project/hadoop-hdfs-httpfs: The patch generated 2 new + 8 unchanged - 5 fixed = 10 total (was 13)  |
| +1 :green_heart: |  mvnsite  |   0m 35s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  shadedclient  |  17m 17s |  patch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 23s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m 59s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   5m 54s |  hadoop-hdfs-httpfs in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 35s |  The patch does not generate ASF License warnings.  |
|  |   |  77m 19s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1737 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux b7ef34bafb31 4.15.0-70-generic #79-Ubuntu SMP Tue Nov 12 10:36:11 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 19512b2 |
| Default Java | 1.8.0_222 |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/3/artifact/out/diff-checkstyle-hadoop-hdfs-project_hadoop-hdfs-httpfs.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/3/testReport/ |
| Max. process+thread count | 594 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-httpfs U: hadoop-hdfs-project/hadoop-hdfs-httpfs |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/3/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java [hunk 2]
```diff
@@ -129,20 +127,14 @@ public void testVarResolutionAndSysProps() {
...
-    out.close();
-    InputStream is = new ByteArrayInputStream(writer.toString().getBytes());
-    conf = new Configuration(false);
-    ConfigurationUtils.load(conf, is);
-    assertEquals(conf.size(), 1);
-    assertEquals(conf.get("key1"), "val1");
+    ConfigurationUtils.load(conf,
+        Thread.currentThread()
+            .getContextClassLoader().getResource(TESTFILE).openStream());
+    assertEquals(conf.size(), 2);
```
Comment (goiri) [inline hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java]: These asserts are revert (also 132).

File: hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java [hunk 3]
```diff
@@ -44,11 +44,12 @@ public void constructors() throws Exception {
...
 
-  @Test(expected = IOException.class)
-  public void constructorsFail3() throws Exception {
-    InputStream is = new ByteArrayInputStream("<xonfiguration></xonfiguration>".getBytes());
+  @Test
+  public void constructors3() throws Exception {
+    InputStream is = new ByteArrayInputStream("<xonfiguration><property name=\"key1\" value=\"val1\"/></xonfiguration>".getBytes());
     Configuration conf = new Configuration(false);
     ConfigurationUtils.load(conf, is);
+    assertEquals(conf.get("key1"), "val1");
```
Comment (goiri) [inline hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java]: Reverse (expected first)

**Comment by goiri** [comment]
Let's fix the checkstyle issues.

File: hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java (hunk 3 above)
Comment (iwasakims) [reply hadoop-hdfs-project/hadoop-hdfs-httpfs/src/test/java/org/apache/hadoop/lib/util/TestConfigurationUtils.java]: fixed the order or arguments of assertEquals.

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 2 new or modified test files.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  21m  5s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 24s |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   0m 21s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 36s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  14m 45s |  branch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 22s |  trunk passed  |
| +0 :ok: |  spotbugs  |   0m 48s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 46s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 23s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 18s |  the patch passed  |
| -0 :warning: |  checkstyle  |   0m 14s |  hadoop-hdfs-project/hadoop-hdfs-httpfs: The patch generated 2 new + 8 unchanged - 5 fixed = 10 total (was 13)  |
| +1 :green_heart: |  mvnsite  |   0m 27s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  shadedclient  |  15m  6s |  patch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 19s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m 45s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 26s |  hadoop-hdfs-httpfs in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 28s |  The patch does not generate ASF License warnings.  |
|  |   |  63m 16s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1737 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux 394666e0daa1 4.15.0-70-generic #79-Ubuntu SMP Tue Nov 12 10:36:11 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 6d970d8 |
| Default Java | 1.8.0_222 |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/4/artifact/out/diff-checkstyle-hadoop-hdfs-project_hadoop-hdfs-httpfs.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/4/testReport/ |
| Max. process+thread count | 617 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-httpfs U: hadoop-hdfs-project/hadoop-hdfs-httpfs |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/4/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 28s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 2 new or modified test files.  |
||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  21m 17s |  trunk passed  |
| +1 :green_heart: |  compile  |   0m 23s |  trunk passed  |
| +1 :green_heart: |  checkstyle  |   0m 18s |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   0m 35s |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  15m  2s |  branch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 20s |  trunk passed  |
| +0 :ok: |  spotbugs  |   0m 40s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m 37s |  trunk passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   0m 21s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 13s |  hadoop-hdfs-project/hadoop-hdfs-httpfs: The patch generated 0 new + 8 unchanged - 5 fixed = 8 total (was 13)  |
| +1 :green_heart: |  mvnsite  |   0m 26s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  shadedclient  |  14m 55s |  patch has no errors when building and testing our client artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 18s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m 44s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   4m 28s |  hadoop-hdfs-httpfs in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 28s |  The patch does not generate ASF License warnings.  |
|  |   |  63m 12s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.4 Server=19.03.4 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1737 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux 64101692556f 4.15.0-70-generic #79-Ubuntu SMP Tue Nov 12 10:36:11 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 6d970d8 |
| Default Java | 1.8.0_222 |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/5/testReport/ |
| Max. process+thread count | 610 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs-httpfs U: hadoop-hdfs-project/hadoop-hdfs-httpfs |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1737/5/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by iwasakims** [comment]
Thanks for the review, @goiri.

**Pull request merged**